### PR TITLE
chore: absolute imports (embed)

### PR DESCRIPTION
**Pull request by brunodesde1987**
**Ticket: [TA-2936 [Embed] Set up absolute imports and replace relative paths](https://gr4vy.atlassian.net/browse/TA-2936)**

### CODE REVIEW

File: packages/embed/package.json [hunk 1]
```diff
@@ -32,7 +32,8 @@
     "form-napper": "^2.0.0"
   },
   "devDependencies": {
-    "@types/applepayjs": "^3.0.2"
+    "@types/applepayjs": "^3.0.2",
+    "eslint-plugin-import": "^2.26.0"
```
Comment (luca-gr4vy) [inline packages/embed/package.json:36]: do things work without this? It's already defined in the main `package.json`, in theory it should pick it up from the root

**Comment by luca-gr4vy** [comment]
@brunodesde1987 getting this when running in dev mode (`yarn dev`), I can help debug later in case

Screenshot 2022-11-30 at 09 57 57

**Comment by brunodesde1987** [comment]
> @brunodesde1987 getting this when running in dev mode (`yarn dev`), I can help debug later in case
> 
> Screenshot 2022-11-30 at 09 57 57

@luca-gr4vy 
Nice catch.
This seems to be an old issue. It's happening on the `main` branch too.
image

File: packages/embed/package.json (hunk 1 above)
Comment (brunodesde1987) [reply packages/embed/package.json]: It seems to work fine.
Done.

**Review by luca-gr4vy — APPROVED** [review]

**Comment by gr4vy-code** [comment]
:rocket: PR was released in [`v2.15.0`](https://github.com/gr4vy/gr4vy-embed/releases/tag/v2.15.0) :rocket:

**Pull request merged**